services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +43,47 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
+  if (!event) {
+    return ERR_CODE_INVALID_QUEUE_MSG;
+  } else if (!thermalMgrQueueHandle) {
+    return ERR_CODE_INVALID_STATE;
+  }
+  BaseType_t errCode = xQueueSend(thermalMgrQueueHandle, event, 0);
+  if (errCode == errQUEUE_FULL) {
+    return ERR_CODE_QUEUE_FULL;
+  }
   return ERR_CODE_SUCCESS;
 }
 
 void osHandlerLM75BD(void) {
   /* Implement this function */
+  thermal_mgr_event_t interrupt = { THERMAL_MGR_EVENT_OS_INTERRUPT };
+  error_code_t errCode = thermalMgrSendEvent(&interrupt);
+  if (errCode) {
+    printConsole("%d\n", errCode);
+  }
```

Comment:
you should not be printing from an ISR. If there is an error its fine to ignore it for the purpose of this onboarding, but on our actual codebase we have specific logging functions meant to be used within an ISR